docs/contributing/pull-requests/index.md

```diff
@@ -115,33 +115,16 @@ These reviews are required for the changes to be merged to `master`.
 You can tag additional teams or individuals for review as you see fit, including `@dept-design` for
 any design changes.
 
+</Bitwarden>
+
 ### Opening the PR for review
 
 As its name implies, marking a PR as "Ready for Review" indicates that you are ready for all
 assigned teams to review it. If the changes are still in progress, leave the PR in `Draft` status.
 Doing this ensures that reviewers can act on the "Ready for Review" as their signal to begin the
 review process without further notification.
 
-### Addressing feedback
-
-It is likely that you will receive some feedback on your PR. You should see this as a positive thing
--- it signifies a healthy and thorough review process and an organizational commitment to code
-quality. You may receive [comments](./code-review.md#comment) or a
-[request for changes](./code-review.md#request-changes). You are encouraged to engage in
-conversation on the PR to discuss a solution, but if any strong conflicting opinions arise it is
-often best to move the conversation to a synchronous format to avoid any misunderstanding.
-
-When any necessary changes have been made, you should address the comments or request for changes by
-responding in the PR conversation thread. You are not responsible for resolving the conversation --
-that is the prerogative of the reviewer, to ensure that they agree that the question or concern has
-been addressed.
-
-**When you are ready for a reviewer to revisit your changes, you should request a re-review.** This
-will notify the reviewer and ensure a prompt response.
-
-</Bitwarden>
-
-## Reviewing the pull request
+## Managing the review process
```

Comment:
I suggest this section is moved to or integrated with `code-review.md`. It describes the expectations the company has about the review process, including expectations of the reviewer, whereas the rest of this page is more of a how-to guide for the PR author. This change in tone and focus is probably the source of the confusion.

You could mention it briefly here, like "You should receive a review or at least first contact from the reviewer within 2 business days of marking the PR as ready for review. If you don't hear anything within this time, it is acceptable (and expected) that you reach out to the reviewer via Slack." Then you could have a callout, e.g. "See [Code Review] for more information about the code review process."

Reply:
I agree it's confusing, and I've struggled with the separation between the "Pull Requests" and "Code Review" content for a while, and with each change I make in this area.

Do you think it would be more helpful to extract all of the review content (from the author and the reviewer perspective) into the "Code Review" page?  It feels like the author / reviewer separation might be artificial and make the documentation confusing.

Reply:
I think that would also work - this page steps you through the process of branching, committing, and submitting the PR, and then you direct the reader to the Code Review page for next steps.

Reply:
I've taken your suggestion to extract that section into the "Code Review" page.  I think it reads much more clearly now - thank you so much for the suggestion!